class-plugin.php

```diff
@@ -33,6 +39,28 @@ public function action_admin_init() {
 
 		register_setting( 'general', 'hm_gtm_id', 'sanitize_text_field' );
 
+		// multisite field below
+		if ( ! is_multisite() ) {
+			return;
+		}
+
+		// add settings for primary blog
+		if ( SITE_ID_CURRENT_SITE === get_current_blog_id() ) {
+
+			// add settings field
+			add_settings_field( 'hm_network_gtm_id_field', esc_html__( 'Network Container ID', 'hm_gtm' ), array(
```

Comment:
I think it makes more sense to have this under the network settings page eg. `wp-admin/network/settings.php`